lighthouse-core/report/html/renderer/category-renderer.js

```diff
@@ -193,16 +190,17 @@ class CategoryRenderer {
    */
   renderAuditGroup(group) {
     const groupEl = this.dom.createElement('div', 'lh-audit-group');
-    const summaryEl = this.dom.createChildOf(groupEl, 'div');
-    const summaryInnerEl = this.dom.createChildOf(summaryEl, 'div', 'lh-audit-group__summary');
-    const headerEl = this.dom.createChildOf(summaryInnerEl, 'div', 'lh-audit-group__header');
 
+    const auditGroupHeader = this.dom.createElement('div', 'lh-audit-group__header');
+
+    this.dom.createChildOf(auditGroupHeader, 'span', 'lh-audit-group__title')
+      .textContent = group.title;
     if (group.description) {
-      const auditGroupDescription = this.dom.createElement('div', 'lh-audit-group__description');
-      auditGroupDescription.appendChild(this.dom.convertMarkdownLinkSnippets(group.description));
-      groupEl.appendChild(auditGroupDescription);
+      const descriptionEl = this.dom.convertMarkdownLinkSnippets(` — ${group.description}`);
```

Comment:
what's the i18n call on these?

Reply:
idk. @exterkamp

Reply:
This seems fine for LTR.  But seems like a landmine for RTL.  But it actually seems to work itself out: 
![image](https://user-images.githubusercontent.com/6392995/56312506-1fef0680-6105-11e9-8f28-a469f5edc320.png)

So LGTM?  I think that we need to keep this in mind for an eventual RTL rework, but looks fine for now.  Put in an i18n TODO here though to be safe I'd say.

Reply:
what if we avoid at least the string concatenation and let the DOM handle it by also having the dash in a `<span>` in addition to the title and description spans?

(that should also help with the tests a bit)

Reply:
done